src/api/app/assets/javascripts/webui2/application.js

```diff
@@ -20,6 +20,7 @@
 //= require jquery.flot.resize
 //= require jquery.flot.time
 //= require jquery.flot.stack.js
+//= require jquery.tokeninput
```

Comment:
This jQuery plugin is [not maintained](https://github.com/loopj/jquery-tokeninput) anymore, so we should use something else for the new Bootstrap UI. Sadly, it doesn't seem like there is something like this which is still maintained. We need to either do it ourselves or go with something different...